react-app/src/library/services/headset.test.ts

```diff
@@ -45,6 +48,13 @@ describe('HeadsetService', () => {
   });
 
   afterEach(() => {
+    plantronics.removeAllListeners();
+    sennheiser.removeAllListeners();
+    jabraNative.removeAllListeners();
+    yealink.removeAllListeners();
+    cyberacoustics.removeAllListeners();
+    vbet.removeAllListeners();
+    jabra.removeAllListeners();
```

Comment:
I ran the tests originally and saw _a lot_ of issues regarding open listeners and a possible memory leak.  So I added these to remove all the listeners after each test.  Everything is built up again before each test so no test accuracy should be affected but this will handle the listeners issue.